Overridden the Status Bars to a create pet button. Fixed the Reset Pet Command as requested in #96. Linted the file and created a global prettier file for different contributors.

Harry-Hopkinson opened the pull request:
This button now adds the option to create a new pet with the status bar. If a pet playground is not the main view, one is created.
Fixed Status Bar Issues

Harry-Hopkinson commented:
@tonybaloney is the code ready for merging?

tonybaloney commented on src/common/types.ts:
```diff
@@ -47,9 +46,9 @@ export const enum Theme {
 }
 
 export const enum ColorThemeKind {
-	Light = 1,
```
Why has this changed? Please revert unless its related to the PR

Harry-Hopkinson replied:
There were some warnings to do with Camel Case.

Harry-Hopkinson commented:
I need some help with the remove pet command as for some reason, the command is not executing my function that removes the selected pet from the array.

Harry-Hopkinson commented:
@tonybaloney In this PR I have implemented the following features.
1. I have overriden the status bear to a create pet button.
2. Fixed the reset pet command that was not working.
3. Save the Pet State after resetting pets.
4. Linted the Files using Prettier.
5. Created a prettier config so contributors, whom use prettier can also lint with the same settings as others.
6. Changed enums to comply with Camel Case warnings.

Harry-Hopkinson commented:
@tonybaloney this should be ready for review!

Harry-Hopkinson commented:
I have tested this code extensively and everything works perfectly 😺

tonybaloney commented:
Thanks for the contribution. I like how the button has a dual-purpose so it doesn't clutter the status bar.

Normally, I'd ask for the Linting changes to be a separate PR as they're unrelated to the change, but as you've put in a lot of time into this PR already, I'll just merge the whole set.
If you're considering contributing in future, I do [strongly recommend keeping each PR to a single, atomic change.](https://developers.google.com/blockly/guides/modify/contribute/write_a_good_pr#keep-it-small)

Harry-Hopkinson commented:
Okay thanks. I will take that into account for future PR's.
